cg/meta/upload/nipt/nipt.py

```diff
@@ -47,14 +47,14 @@ def set_dry_run(self, dry_run: bool) -> None:
         LOG.info("Set dry run to %s", dry_run)
         self.dry_run = dry_run
 
-    def target_reads(self, case_id: str) -> float:
+    def expected_reads(self, case_id: str) -> float:
         """Return the target reads of the case"""
         case_obj: models.Family = self.status_db.family(case_id)
         application_version = self.status_db.ApplicationVersion.query.filter(
             models.ApplicationVersion.id == case_obj.links[0].sample.application_version_id
         ).first()
```

Comment:
I think this SQL query and filter does not belong in nipt.py. It should be in the StatusDBAPI along with unit tests. Nipt.py should only use a "get_application_version(application_version_id=case_obj.links[0].sample.application_version_id
        ).first())" function or something similar otherwise it "knows" to much about the database making this very fragile and also hard to maintain and test.
And a type hint